styles/globals.css

```diff
@@ -2,6 +2,34 @@
 @tailwind components;
 @tailwind utilities;
 
+.light {
+  --blue600: 30, 136, 229; /* blue600 */
+  --blue700: 25, 118, 210;/* blue700 */
+  --grey900: 33, 37, 41;/* grey900 */
+  --grey800: 52, 58, 64;/* grey800 */
+  --grey700: 73, 80, 87;/* grey700 */
+  --grey650: 107, 115, 123;/* grey650 */
+  --grey600: 108, 117, 125;/* grey600 */
+  --grey500: 173, 181, 189;/* grey500 */
+  --grey400: 206, 212, 218;/* grey400 */
+  --grey300: 222, 226, 230;/* grey300 */
+  --grey200: 233, 236, 239;/* grey200 */
+  --grey100: 248, 249, 250;/* grey100 */
+  --red200: 239, 154, 154;/* red200 */
+}
+
+.dark {
+  --hyperlinkDark: 82, 156, 236;
+  --backgroundBlueDark: 78, 134, 220;
+  --LabelHeadingsDark: 20, 20, 25;
+  --grey650Dark: 94, 102, 109; /* not used? */
+  --globalBorderGrey: 78, 86, 94;
+  --grey300Dark: 35,39,43;
+  --boxscoreBorderGrey:  45, 48, 51;
+  --backgroundGrey100: 33, 37, 41;
+  --white:  248, 249, 250;
+}
+
```

Comment:
Okay so while this works we can do wayyyyyy better. The drawback to this approach is that we have two completely different naming schemes for the variables as well as two completely different naming schemes for the tailwind classes. The way things are now, we might as well just use tailwinds colors and define everything with `text-black dark:text-white` since we aren't benefiting from this in any way.

*Ideally* we should have one unique set of variable names where the value for each variable is defined for dark mode and light mode, so when the `light` or `dark` class is present in our root node we should be using that set of defined variables.

Additionally, the names present in the dark mode are minimal and only seem to be for the cases where they exist which isn't very scalable. So I propose a new set of colors that are more exhaustive and support a wider range of cases that we can just copy over to the portal if needed

Let's make make a more exhaustive set of variables where each type is:
- Deprecated: (i.e. the `--color200`) style variables. These should remain static between dark and light mode.
- Border colors: These should be things like `--color-border-primary` or `--color-border-secondary`. That way we have a more semantic way of writing the color for the case it is being used in.
- Text colors: This includes `--color-text-primary`, `--color-text-secondary`, `--color-text-link`, etc.
- Background colors: For here we will want to include the `--color-bg-primary` and `--color-bg-secondary`

For each semantic color group we should include `-inverted` colors to handle hover states and more. There is more we can do to clean this up but this should be a good first step and I can possibly go in afterwords and clean it up a bit more.